.github/workflows/release_macos.yml

```diff
@@ -25,11 +25,11 @@ jobs:
 
     - name: Install dependencies
       run: |
-        brew install ffmpeg freetype fribidi harfbuzz python@3.11 curl pkg-config nasm create-dmg
+        brew install ffmpeg freetype fribidi harfbuzz python@3.13 curl pkg-config nasm create-dmg
 
     - name: Setup venv
       run: |
-        $(brew --prefix)/opt/python@3.11/libexec/bin/python configure.py
+        $(brew --prefix)/opt/python@3.13/libexec/bin/python configure.py
```

Comment:
Non blocking and not related to MacOS but there is also:
```
.github/workflows/ci_msvc.yml:          python-version: '3.11'
.github/workflows/release_windows.yml:          python-version: '3.11'
```

Reply:
Good catch. Added a separate commit.